website/content/partials/versions.mdx

```diff
@@ -0,0 +1,20 @@
+## Vault Version Guidance
```

Comment:
Use [sentence case for all titles & sub-titles](https://docs.google.com/document/d/1eA5AWPWPbJaWh4cSSrLiQb3FiPFw52rVNqB-Q9YGGYI/edit?usp=sharing)

```suggestion
## Vault Version guidance
```